ublox-short-range/src/wifi/socket.rs

```diff
@@ -410,12 +401,8 @@ where
                         .map_err(|_e| Self::Error::BadLength)?;
                 }
                 IpAddr::V6(ip) => {
-                    workspace = to_string(
-                        &ip,
-                        String::<consts::U1>::new(),
-                        SerializeOptions::default(),
-                    )
-                    .map_err(|_e| Self::Error::BadLength)?;
+                    workspace = to_string(&ip, String::<1>::new(), SerializeOptions::default())
```

Comment:
I know this was made by Mads, but it's probably not a good idea to use `serde_at` for these, as `serde_at` should be considered an internal crate.

An alternative could be to utilize the already implemented `Display` in no-std-net as:
```suggestion
                    workspace
                        .write_fmt(format_args!("{}", ip))
                        .map_err(|_| Self::Error::BadLength)?;
```

With `let workspace = String::<43>::new();` on line 384 above

Reply:
It was made before we had the `write_fmt` function :wink:

Reply:
`write_fmt` is a core library function, so i doubt that ;)